src/new.rs

```diff
@@ -133,10 +117,10 @@ fn create_file<P: AsRef<Path>>(name: P, content: &[u8]) -> Result<()> {
 
     let mut file = try!(OpenOptions::new()
         .write(true)
-        .create(true)
+        .create_new(true)
```

Comment:
I take it this will report an error?  Whats the error look like? Is it good enough for our users?